packages/mdc-menu/simple/mdc-simple-menu.scss

```diff
@@ -53,50 +55,48 @@ $mdc-simple-menu-item-fade-duration: .3s;
     outline: none;
   }
 
+  &--animating-open {
+    display: inline-block;
+    overflow-y: hidden;
+    opacity: 0;
+    transform: scale(0.8);
+    transition: opacity $mdc-simple-menu-fade-duration linear,
+                transform $mdc-simple-menu-scale-duration $mdc-animation-deceleration-curve-timing-function;
+    > .mdc-simple-menu__items {
+      transform: scale(1.25);
+    }
+  }
+
   &--open {
     display: inline-block;
     transform: scale(1);
     opacity: 1;
+    > .mdc-simple-menu__items {
```

Comment:
Is this necessary, or would this be the default style anyway due to the style below under `&__items`?

Reply:
yes. This is the end state that gets applied after animating-open class is added.